services/ui-src/src/components/reports/ModalOverlayReportPage.tsx

```diff
@@ -263,7 +177,7 @@ export const ModalOverlayReportPage = ({
           />
           <DeleteEntityModal
             entityType={entityType}
-            selectedEntity={currentEntity}
+            selectedEntity={selectedEntity}
```

Comment:
You know what? I think we can do this. But to do that we should create a nice-to-have ticket for it. It would touch not just initiatives but transition benchmarks and it might be nice to have a more focused state rework pr.